Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -3,23 +3,23 @@
 using System;
 
 [Serializable]
-public class TimeRenderSettings
+public struct TimeRenderSettings
 {
-	public Color skyColor = Color.white;
-	public Color lightColor = Color.white;
-	public float lightIntensity = 0.5f;
-	public Color fogColor = Color.white;
-	public float fogDensity = 0.05f;
+	public Color skyColor;
+	public Color lightColor;
+	public float lightIntensity;
+	public Color fogColor;
+	public float fogDensity;
```

Comment:
Just 0's. So the default's become less reasonable, but we produce less garbage on updates. I think it's worthwhile.